eva/server/db_api.py

```diff
@@ -76,15 +77,20 @@ def _upload_transformation(self, query: str) -> str:
          the content into a string.
         """
         if 'UPLOAD' in query:
-            file_path = query.split()[2][1:-1]
-            dst_path = query.split()[-1][1:-2]
+            query_list = query.split()
+            file_path = query_list[2][1:-1]
+            dst_path = os.path.basename(file_path)
             with open(file_path, "rb") as f:
                 bytes_read = f.read()
                 b64_string = str(base64.b64encode(bytes_read))
                 query = 'UPLOAD PATH ' + \
```

Comment:
And also maybe use `'{}'.format` to clean it up

Reply:
Resolved.